autograder/plagiarism_detection/plagiarism_detection.py

```diff
@@ -53,3 +99,9 @@ def getSimilarity(a, b):
         if a[i] == b[i]:
             numberSame = numberSame + 1
     return numberSame / minLen
+
+
+# first, find number of each token type - done
```

Comment:
These comments should be somewhere else. For example, in your repo as issues or at least as #TODO: style comments.